comp/api/api/def/component.go

```diff
@@ -41,6 +41,24 @@ type endpointProvider struct {
 	handler http.HandlerFunc
 }
 
+// AuthorizedSet is a type to store the authorized config options for the config API
+type AuthorizedSet map[string]struct{}
+
+// AuthorizedConfigPathsCore is the the set of authorized config keys authorized for the
+// config API.
+var AuthorizedConfigPathsCore = buildAuthorizedSet(
+	"api_key", "site", "dd_url", "logs_config.dd_url",
+	"additional_endpoints", "logs_config.additional_endpoints",
```

Comment:
We added `"additional_endpoints", "logs_config.additional_endpoints",` but there seems to be no code for handing the `"additional_endpoints"` key sync, is that intended?

Reply:
We don't need additional code to manage that because it maps automatically with seamless unmarshaling.